netket/sampler/metropolis.py

```diff
@@ -525,3 +525,20 @@ def MetropolisHamiltonian(hilbert, hamiltonian, *args, **kwargs) -> MetropolisSa
 
     rule = HamiltonianRule(hamiltonian)
     return MetropolisSampler(hilbert, rule, *args, **kwargs)
+
+def MetropolisGaussian(hilbert, sigma, *args, **kwargs) -> MetropolisSampler:
+    r"""This sampler acts on all particle positions simultaneously
+    and proposes a new state according to a Gaussian distribution
+    with width sigma.
+    Args:
+       hilber: The continuous Hilbert space
+       sigma: The width of the Gaussian proposal distribution
+       n_chains: The number of Markov Chain to be run in parallel on a single process.
+       sweep_size: The number of exchanges that compose a single sweep.
+                   If None, sweep_size is equal to the number of degrees of freedom (n_visible).
+    """
+
```

Comment:
maybe check that the hilbert space `isinstance(hlibert, AbstractParticle)` and throw an error otherwise?

Reply:
`ValueError("This sampler only works for Continuous Hilbert spaces, my dear and beloved user")`